meilisearch/src/routes/indexes/settings.rs

```diff
@@ -137,10 +137,8 @@ macro_rules! make_setting_route {
                 let settings = settings(&index, &rtxn, meilisearch_types::settings::SecretPolicy::HideSecrets)?;
 
                 debug!(returns = ?settings, "Update settings");
-                let mut json = serde_json::json!(&settings);
-                let val = json[$camelcase_attr].take();
 
-                Ok(HttpResponse::Ok().json(val))
+                Ok(HttpResponse::Ok().json(dbg!(settings.$attr)))
```

Comment:
Using `serde_json::json!` to pre-serialize the settings will cast the f32 in distribution into f64, changing the value of the floats.
We now avoid making a double serialization.

related to https://github.com/meilisearch/meilisearch/issues/4594